### ISSUE TITLE
demo.py: 'SupportedSRS' object is not iterable 

### ISSUE THREAD

**Issue by DirkThalheim** [mapproxy/mapproxy#430]
After Updating to MapProxy 1.12.0 I'm getting errors when calling the demo application for certain WMS source configurations using supported_srs option.

Configuration:
```sources:
  my_source:
    type: wms
    req:
      # ...
    supported_srs: ['EPSG:4326','EPSG:3857']
    coverage:
      bbox: [247087, 5227094, 947317, 6131781]
      srs: 'EPSG:25832'
```
Error Report:
```   File ".../python2.7/site-packages/mapproxy/service/demo.py", line 144, in layer_srs
     for supported_srs in map_layer.supported_srs:
 TypeError: 'SupportedSRS' object is not iterable at line 59 column 39 in file .../python2.7/site-packages/mapproxy/service/templates/demo/demo.html
```

It works again when changing line 144 in demo.py to
```
for supported_srs in map_layer.supported_srs.supported_srs:
```

**Comment by sdhiscocks** [comment]
I've also got a proposed fix in #420

**Comment by leader80** [comment]
is it going to be part of next release or already integrated in current one?
